presto-spi/src/main/java/com/facebook/presto/spi/block/BlockUtil.java

```diff
@@ -208,4 +209,11 @@ static boolean arraySame(Object[] array1, Object[] array2)
         }
         return true;
     }
+
+    public static boolean rawPositionInRange(int position, int offset, int positionCount)
```

Comment:
nit: perhaps, rename `position` to `rawPosition` here and in other places where it is used